src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -43,6 +44,16 @@
     <File Include="$(PlaceholderFile)">
       <TargetPath>ref/netstandard1.0</TargetPath>
     </File>
+    <!-- Symbols -->
+    <ArchitectureSpecificNativeSymbol Include="@(ArchitectureSpecificNativeFileAndSymbol -> '%(RelativeDir)PDB\%(FileName).pdb')" />
+    <ArchitectureSpecificNativeSymbol Include="@(ArchitectureSpecificLibFile -> '%(RelativeDir)PDB\%(FileName).pdb')" />
+    <ArchitectureSpecificNativeSymbol Include="@(ArchitectureSpecificToolFile -> '%(RelativeDir)PDB\%(FileName).pdb')" />
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.dll" />
```

Comment:
Right now this is inconsistent between Windows and other platforms, but symbol packages should actually include the dll's since they're also indexed on MyGet's symbol server. I'm pretty sure this applies to exe's too. In the future, we'll likely start adding binaries to the xplat symbol packages too once we have a service up that lets you grab only the xplat symbols that you need.